src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -196,7 +196,7 @@ form.comment {
     display: none;
 }
 
-.commentable {
+.commentable, .clickable {
```

Comment:
use `<a href="#">` instad.

Reply:
This adds an underline to the element.

Reply:
Then use `<a>` without href

Reply:
<a> without href does not give cursor pointer. I would still need to modify css to add an a:hover selector.

Reply:
https://getbootstrap.com/docs/4.4/utilities/text/#text-decoration ?

Reply:
Hah ok seems like a lot of ceremony when we already have a selector for this.

Reply:
Having clickable things be `<a>` or `<button>` is a bit semantically cleaner.

Reply:
So is there a reason we don't do that for comments?

Reply:
We do. 
```
<a href="#" id="Azure.Messaging.EventHubs.EventProcessorClient.ProcessEventAsync" class="name commentable">ProcessEventAsync</a>
```

Reply:
So why is commentable defined in site.scss?

Reply:
Don't know 😕

Reply:
Ok I will delete it as a POC.